independent-projects/resteasy-reactive/server/processor/src/main/java/org/jboss/resteasy/reactive/server/processor/ResteasyReactiveDeploymentManager.java

```diff
@@ -79,8 +79,9 @@ public static ScanStep start(IndexView nonCalculatingIndex) {
     public static class ScanStep {
         final IndexView index;
         int inputBufferSize = 10000;
+        int outputBufferSize = 8192;
```

Comment:
So this is used when outside of Quarkus? If so, do we really want the value to not be consistent with Quarkus?

Reply:
Right